homeassistant/components/emoncms/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "id": "id",
+          "url": "Server Hostname",
+          "api_key": "API Key"
```

Comment:
Please use references to the common strings. Use the data_description field to add more context to the fields

Reply:
done